cleanups and minor improvements

muhomorr opened the pull request.

muhomorr committed:
remove copy of default app icon

Default app icon is used when no icon is specified.

muhomorr committed:
remove android:hardwareAccelerated="true" from AndroidManifest

android:hardwareAccelerated is true by default.

muhomorr committed:
remove DEVICE_INITIALIZATION_WIZARD intent action from WelcomeActivity

It's been deprecated long ago and is no longer used by the OS.

muhomorr committed:
remove Int.MAX_VALUE unix time limit

Unix timestamps are 64-bit on Android.

muhomorr committed:
remove deprecated and unnecessary hideSystemBars() from FinishActions

SystemBarHelper.hideSystemBars() applies to the current activity, which is finishing when
hideSystemBars() is called.

muhomorr committed:
add default values to SetupWizardActivity constructor parameters

muhomorr committed:
remove unused setupcompat and setupdesign prebuilts

muhomorr committed:
prevent potential appContext initialization order issue

ContentProviders are initialized before Application.onCreate(), but after attachBaseContext().

muhomorr committed:
make GrapheneOS logo black or white

colorPrimary can be any color, depending on theme configuration.

textColorPrimary is black with light theme and white with dark theme.

jiteshsingh commented on app/src/main/java/app/grapheneos/setupwizard/App.kt:
```diff
@@ -1,10 +1,12 @@
-        super.onCreate()
-        appContext = this
+
+    override fun attachBaseContext(base: Context) {
```
is `attachBaseContext` more reliable and gets always called?

muhomorr replied:
Both onCreate() and attachBaseContext() are reliable, but Application.onCreate() is called after ContentProvider initialization.

jiteshsingh committed:
toggles location when the container view is clicked

- container view = checkbox + description
- also adds visual indicators for the click

jiteshsingh commented on app/src/main/java/app/grapheneos/setupwizard/view/activity/LocationActivity.kt:
```diff
@@ -23,6 +24,9 @@ class LocationActivity : SetupWizardActivity(
 
     override fun setupActions() {
         enabled.setOnClickListener { LocationActions.setEnabled(enabled.isChecked) }
+        findViewById<TextView>(R.id.enabled_desc).setOnClickListener {
```
in this case, better to attach listener on the container view (the linear layout)?
and also provide visual indications of the click?

pushed the change here; you can rebase. https://github.com/jiteshsingh/platform_packages_apps_SetupWizard/commits/location_click_container/

jiteshsingh replied:
or I can merge my change later, either way.
